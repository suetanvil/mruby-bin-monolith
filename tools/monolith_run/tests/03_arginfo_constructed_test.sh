#!/bin/bash

set -e

./print_arginfo a b c
