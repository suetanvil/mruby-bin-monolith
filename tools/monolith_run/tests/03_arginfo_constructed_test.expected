$0: ./print_arginfo
ARGV[0]: a
ARGV[1]: b
ARGV[2]: c
MC::ExePath: $ROOT/tests/print_arginfo
MC::ProgPath: $ROOT/tests/print_arginfo
MC::IsConstructed: true
Done.
